src/widget/extra_info.h

```diff
@@ -0,0 +1,28 @@
+#ifndef WIDGET_EXTRA_INFO_H
+#define WIDGET_EXTRA_INFO_H
+
+  //#include "sidebar.h"
+
+typedef struct {
+    int value;
+    int target;
+} objective;
+
+typedef struct {
+    int height;
+    int game_speed;
+    int unemployment_percentage;
+    int unemployment_amount;
+    objective culture;
+    objective prosperity;
+    objective peace;
+    objective favor;
+    objective population;
+} extra_info;
```

Comment:
If structs are not used anywhere else besides the corresponding `.c` file, put them in the `.c` file to not pollute the global namespace with them.

Reply:
totally agree with ya, unfortunately this extra_info struct is a field in the data struct in sidebar.c 

https://github.com/bvschaik/julius/blob/master/src/widget/sidebar.c#L142

```
static struct {
    time_millis slide_start;
    int progress;
    int focus_button_for_tooltip;
    struct {
        int height;
        int game_speed;
        int unemployment_percentage;
        int unemployment_amount;
        objective culture;
        objective prosperity;
        objective peace;
        objective favor;
        objective population;
    } extra_info;
} data;
```

however it looks like the old sidebar.c doesn't use extra_info anymore after i split it all up, so ill probably be able to remove the extra_info field from the old data struct.